package.json

```diff
@@ -14,7 +14,7 @@
     "eslint",
     "config"
   ],
-  "author": "Joe Lencioni",
+  "author": "Brigade Group, Inc.",
```

Comment:
It seems they changed the name a while back from Brigade Media to Brigade Group. I'll bring it up with whoever controls our ToS, thanks!